druntime/src/core/sys/posix/time.d

```diff
@@ -200,8 +200,21 @@ version (linux)
 {
     struct timespec
     {
+        version (CRuntime_Musl) {
+            this(time_t tv_sec_, c_long tv_nsec_) @safe pure nothrow
+            {
+                tv_sec = tv_sec_;
+                tv_nsec = tv_nsec_;
+            }
+        }
```

Comment:
Preferred to be without constructors to avoid making in non-POD.

Reply:
https://dlang.org/spec/struct.html#POD says that *copy* constructors are not allowed on a POD, plain constructors are fine. `__traits(isPOD)` does return `true` for this sample and allowing non-copy constructors has been a conclusion in an old [discussion](https://forum.dlang.org/post/kbmblk$2jg8$1@digitalmars.com) as well.

Reply:
@sertonix - D has supported named arguments since 2023.

https://dlang.org/spec/expression.html#argument-list

https://dlang.org/spec/struct.html#struct-literal

That can be changed to:
```diff
             immutable tv_sec = toUnixTime!(typeof(timespec.tv_sec))();
             immutable fracHNSecs = removeUnitsFromHNSecs!"seconds"(_stdTime - 621_355_968_000_000_000L);
             immutable tv_nsec = cast(typeof(timespec.tv_nsec))convert!("hnsecs", "nsecs")(fracHNSecs);
-            return timespec(tv_sec, tv_nsec);
+            return timespec(tv_sec: tv_sec, tv_nsec: tv_nsec);
         }
 
         @safe unittest
```

Reply:
I now used unnamed bitfields instead